installer/lib/mix/tasks/phx.new.ex

```diff
@@ -112,7 +113,16 @@ defmodule Mix.Tasks.Phx.New do
 
   @impl true
   def run([version]) when version in ~w(-v --version) do
-    Mix.shell().info("Phoenix v#{@version}")
+    version_and_revision =
+      with revision when revision != "" <- @revision,
```

Comment:
I don't believe we can yet support `with` based on our Elixir version support. Can you swap it?

Reply:
> I don't believe we can yet support `with` based on our Elixir version support. Can you swap it?

`with` was introduced in v1.2.0
https://hexdocs.pm/elixir/1.2.0/Kernel.SpecialForms.html#with/1